### DISCUSSION TITLE
pip install zeep

### DISCUSSION THREAD

**Discussion by 1kohm** [PiotrMachowski/Home-Assistant-custom-components-Burze.dzis.net#12] [Q&A] [answered]
Hi, 

I want to use your module on my HA running on RPI4 but facing an issue with home assistant cannot find module burze_dzis_net. I assume that it's due to missing "zeep" package. 
I have python and python3 in following versions:

`@DevHass:~ $ python --version
Python 2.7.16
@DevHass:~ $ python3 --version
Python 3.7.3`

My question is, how should I safety install pip manager? Should it be done from OS console (Raspbian) with command:

`apt install python-pip`

or from HA terminal?

I ask as I don't want to brick my HA system :)

Pozdrawiam :)

**Comment by PiotrMachowski** [comment, marked as answer]
Hi!

How have you installed this integration? I strongly suggest using [HACS](https://hacs.xyz/).

What type of installation do you have? You can check it here: [![Open your Home Assistant instance and show your Home Assistant version information.](https://my.home-assistant.io/badges/info.svg)](https://my.home-assistant.io/redirect/info/) (`<HA URL>/config/info`).

If you use *Home Assistant Core* in Python virtual environment then you have to open it, install pip if it is not available (`sudo apt install python-pip3`) and install zeep (`pip3 install zeep`).

If you don't use *Home Assistant Core* (and I think that you don't - because you have HA terminal) then HA should install this dependency by itself after a restart. You have to install this integration, restart HA, configure this integration and restart HA again.

**Reply by 1kohm** [reply]
Hi, 
I'm running 
`
<html>
<body>


Version | core-2021.5.3
-- | --
Installation Type | Home Assistant Supervised


</body>
</html>`

I did reinstall your addon via HACS, then did RPI4 reboot, next configuration and HA reload. 
That solved my issue, so thanks for support and great work! 

Pozdrawiam!
